styles/buyer.css

```diff
@@ -0,0 +1,86 @@
+*{
+    padding: 0%;
+    margin: 0%;
+}
+ html{
+     font-size: 16px;
+ }
+
+
+.navBar{
```

Comment:
change rulset to nav after changing the div element in HTML file

Reply:
DONE